### ISSUE TITLE
If bored.. rename/clean globals

### ISSUE THREAD

**Issue by onkelandy** [Morg42/MultiDevice#11]
ITEM_ATTR_CYCLE              = 'md_read_cycle' 
ITEM_ATTR_READ_INIT          = 'md_read_initial'        # trigger read on initial connect
ITEM_ATTR_GROUP              = 'md_read_group'          # trigger read with read group <foo>
ITEM_ATTR_READ_GRP           = 'md_read_group_trigger'

is not very intuitive though most likely noone will ever care ;)
Still suggest:
ITEM_ATTR_READ_CYCLE              = 'md_read_cycle' 
ITEM_ATTR_READ_INIT          = 'md_read_initial'        # trigger read on initial connect
ITEM_ATTR_READ_GROUP              = 'md_read_group'          # trigger read with read group <foo>
ITEM_ATTR_READ_TRIGGER          = 'md_read_group_trigger'

..and a bonus ;)
PLUGIN_ATTR_CLEAN_STRUCT     = 'clean_structs' 
-> 
PLUGIN_ATTR_CLEAN_STRUCTS     = 'clean_structs' 

CMD_ATTR_PARAM_VALUES should be obsolete, right? All references in imports, docue and e.g. line 588 in MD_Command.py might be killed? Inline docu of MD_Command_JSON needs update.

**Comment by Morg42** [comment]
The first "inconsistence" stems from the wish to not unnessecarily "enlargen" the code lines... though basically you're right ;)

So - (have been waiting on writing this forever - ) won't fix ;)

Bonus is fixed, CMD_ATTR_PARAM_VALUES removed.

Inline docu of JSON will be updated when JSON is fixed (has issues in kodi atm)
